model/jvm/src/main/scala/hmda/model/edits/EditMetaDataLookup.scala

```diff
@@ -0,0 +1,42 @@
+package hmda.model.edits
+
+import hmda.model.ResourceUtils
+import com.github.tototoshi.csv.CSVParser.parse
+
+object EditMetaDataLookup extends ResourceUtils {
+  val values: Seq[EditMetaData] = {
+    val lines = resourceLines("/edit-metadata.txt", "iso-8859-1")
+
+    lines.drop(1).map { line =>
+      val values = parse(line, '\\', ',', '"').getOrElse(List())
+      val category = values(0)
+      val editType = values(1)
+      val fieldNames = values(2)
+      val editNumber = values(3)
+      val editDescription = values(4)
+      val explanation = values(5)
+      val userFriendlyEditDescription = values(6)
+
+      EditMetaData(
+        category,
+        editType,
+        fieldNames,
+        editNumber,
+        editDescription,
+        explanation,
+        userFriendlyEditDescription
+      )
+
+    }.toSeq
+  }
+}
+
+case class EditMetaData(
+  category: String,
+  editType: String,
+  fieldNames: String,
```

Comment:
I noticed in the file that `fieldNames` only lists one field for each edit, though most edits have several relevant fields. (for example, `V275` depends on `applicationDate` and `actionTakenDate`, but only "Action Taken Date" is listed in the file.)

This isn't relevant for this PR, but it seems like it will become relevant soon.